travis/install-jhipster.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ev
+
+#-------------------------------------------------------------------------------
+# Choose the repo
+#-------------------------------------------------------------------------------
+export JHIPSTER_REPO=https://github.com/jhipster/generator-jhipster.git
+export JHIPSTER_BRANCH=master
```

Comment:
@PierreBesson : for example, in `spring-boot-2` branch of `jhipster-dependencies`, we can put here `spring-boot-2` branch for `generator-jhipster`
So it's for testing a sync branch with one upstream. I don't know if I'm clear enough ?

Reply:
Awesome